lib/wakes/configuration.rb

```diff
@@ -15,9 +15,13 @@ def self.configure
 
   class Configuration
     attr_accessor :enabled
+    attr_accessor :ga_profiles
 
     def initialize
       @enabled = true
+      @ga_profiles = {
+        'default' => ENV['GOOGLE_ANALYTICS_PROFILE_ID']
```

Comment:
@chrismaddocks what do you think about changing the default key from `'default'` to `ENV['DEFAULT_HOST']`?

Then you could do this in the `Wakes::Location`:

    def google_analytics_profile_id
      Wakes.configuration.ga_profiles[host || ENV['DEFAULT_HOST']]
    end

and then our app initializer would look something like:

    Wakes.configure do |config|
      config.ga_profiles = {
        'www.desiringgod.org' => ENV['GOOGLE_ANALYTICS_PROFILE_ID'],
        'solidjoys.desiringgod.org' => ENV['GOOGLE_ANALYTICS_SJ_PROFILE_ID']
      }
    end

Which is easier to follow.